src/main/initNrfUtilProxyEnv.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+export default () => {
+    const index = process.argv.findIndex(p => p.includes('--proxy-server'));
+    if (index !== -1) {
+        const proxy = `http://${process.argv[index].split('=')[1]}`;
+        process.env.HTTP_PROXY = proxy;
+        process.env.HTTPS_PROXY = proxy;
+        process.env.http_proxy = proxy;
+        process.env.https_proxy = proxy;
+    }
+};
```

Comment:
S: Instead of parsing `argv` yourself, you can use our `argv` module here:

```suggestion
import argv from './argv';

export default () => {
    if (argv['proxy-server']) {
        const proxy = `http://${argv['proxy-server']}`;

        process.env.HTTP_PROXY = proxy;
        process.env.HTTPS_PROXY = proxy;
        process.env.http_proxy = proxy;
        process.env.https_proxy = proxy;
    }
};
```